spec/std/range_spec.cr

```diff
@@ -26,6 +26,14 @@ struct RangeSpecIntWrapper
   end
 end
 
+private def assert_in_delta(expect, actual, delta)
```

Comment:
We have `be close(expected_value, delate)`. See https://github.com/crystal-lang/crystal/blob/master/src/spec/expectations.cr#L231